server/routes/sentry/tasks.py

```
@@ -32,6 +32,38 @@
+        if cursor:
+            cursor.execute(
+                "SELECT preference_value FROM user_preferences WHERE user_id = %s AND preference_key = 'sentry_rca_resources'",
+                (user_id,),
+            )
+            row = cursor.fetchone()
+        else:
+            from utils.db.connection_pool import db_pool
+            from utils.auth.stateless_auth import set_rls_context
+
+            with db_pool.get_admin_connection() as conn:
+                with conn.cursor() as cur:
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)